functions/function_scope.js

```diff
@@ -0,0 +1,16 @@
+/**
+* Topic: divideByThree function does not print
+* Problem instructions: where you should close this function block?
+* Relevant resources:
+* - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_Types
```

Comment:
Add more links like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions